server/api/controllers/forms/post-create-submission.js

```diff
@@ -153,6 +154,71 @@ module.exports = {
         });
       }
 
+      // Send alert email to leaders if setting is turned on
+      if (formRecord[0].alertType !== 'None') {
+        let emailRecipients;
+
+        // get email addresses of the recipients
+        if (formRecord[0].alertType === 'Custom') {
+          emailRecipients = formRecord[0].customAlertRecipients;
+        } else if (
+          ['LIFE Group', 'Lifestage', 'Campus'].includes(
+            formRecord[0].alertType
+          )
+        ) {
+          const recipientsDict =
+            await sails.helpers.forms.getFormAlertRecipients(formId);
+
+          // get the current season leadership teams
+          const today = new Date();
+
+          let fieldSelected;
```

Comment:
do we need to do error handling here like in case somehow there's bad input or smth.

Reply:
i camelized the field selected, does that work in terms of bad input? because the fieldSelected will always be one of the choices of the Alert Types and then camelize that